extensions/wikia/ARecoveryEngine/js/SourcePoint/slotFinder.js

```diff
@@ -1,5 +1,5 @@
 /*global define*/
-define('ext.wikia.aRecoveryEngine.recovery.slotFinder', [
+define('ext.wikia.aRecoveryEngine.sourcePoint.slotFinder', [
```

Comment:
I'd add fallback only for `ext.wikia.aRecoveryEngine.recovery.sourcePoint` due to this change: https://github.com/Wikia/app/pull/12808/files#diff-939a1bcfebcc0e82ae00ac908560e39cR22